### PR TITLE
Add One S docs (without images)

### PR DESCRIPTION
**Pull request by diastremskii**
This PR adds One S docs, but they do need some images for build process (and PCB pics)

### CODE REVIEW

File: docs/keyboards/ergonaut-one-s/build-preparations.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+**Note: this model contains included bed adhesion-improving supports, which should be removed after print.**
+
+| Bottom part placement                                          | Top part placement                                             |
+| -------------------------------------------------------------- | -------------------------------------------------------------- |
+| ![3D Printed Classic Case Bot](/img/one_classic_bot_print.jpg) | ![3D Printed Classic Case Top](/img/one_classic_top_print.jpg) |
+
+**Note: After the print don't forget to remove included supports from the top parts. The image below for a reference:**
+
+![3D Printed Classic Case Supports](/img/one_classic_top_supports.jpg)
```
Comment (diastremskii) [inline docs/keyboards/ergonaut-one-s/build-preparations.md:119]: Is it all right to keep same image for removing supports?

File: docs/keyboards/ergonaut-one-s/build-guide.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+### Clean up
+
+Clean your PCB from flux residues. IPA (isopropyl alcohol) with Q-tips should do the trick.
+
+### Flash firmware
+
+Now it's a good time to flash firmware to a keyboard.
+
+1. Download [firmware](https://github.com/ergonautkb/one-zmk-config/releases/latest/download/ergonaut_one_firmware.zip) and unzip it.
```
Comment (diastremskii) [inline docs/keyboards/ergonaut-one-s/build-guide.md:101]: Needs to be updated when release is added to https://github.com/ergonautkb/one-s-zmk-config

File: docs/keyboards/ergonaut-one-s/build-preparations.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+| **[OPTIONAL]** JST SH1.0 Horizontal 2 Pin PCB Connector | 2   | [Aliexpress](https://aliexpress.com/item/1005005682408443.html?&sku_id=12000033998015849) |
+
+## Case
+
+The case is also an essential part of Ergonaut One S since it provides proper electrical insulation from the environment and provides reliable mounting plates for the switches.
+
+You can choose to use one of three provided variants of the case:
+
+* 3D Printed:
+  * [Modern Case](#3d-printed-modern-case) (Recommended) (The easiest to print)
```
Comment (pashutk) [inline docs/keyboards/ergonaut-one-s/build-preparations.md]:
```suggestion
  * [Modern Case](#3d-printed-modern-case) (Recommended, the easiest to print)
```

Comment (pashutk) [reply docs/keyboards/ergonaut-one-s/build-preparations.md]: Both here and in One docs

**Review by pashutk — APPROVED** [review]

File: docs/keyboards/ergonaut-one-s/build-preparations.md (hunk 3 above)
Comment (diastremskii) [reply docs/keyboards/ergonaut-one-s/build-preparations.md]: Fixed in both places

**Pull request merged**